psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -80,6 +80,7 @@ def scf_compute_energy(self):
         self.initialize_jk(self.memory_jk_)
     else:
         self.initialize()
+    self.iterene = []
```

Comment:
Use a more descriptive variable name, e.g.
```suggestion
    self.iteration_energies = []
```